README.md

```diff
@@ -19,11 +20,12 @@ To set up a virtual environment for this application, follow these steps:
 
 on Windows:
 
-    `cd venv/Scripts/activate`
+` cd venv/Scripts/activate`
 
 on macOS/Linux
 
-    `cd venv/bin/activate`
+`cd venv/bin/activate`
```

Comment:
I think it should be source venv/bin/activate

Reply:
I have done it